<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ARPAP_SpatialReportDialogBase</class>
 <widget class="QDialog" name="ARPAP_SpatialReportDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ARPAP SpatialReport</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabSaveLoad">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabRun">
      <attribute name="title">
       <string>Run</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="3">
        <widget class="QPushButton" name="runButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Run</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="4">
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="pageSelectLayer">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="3" column="1">
            <widget class="QLabel" name="labelInfoOriginLayer">
             <property name="font">
              <font>
               <family>Droid Sans [unknown]</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Select origin layer from combobox</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="labelTitleStep1">
             <property name="font">
              <font>
               <family>Droid Sans [unknown]</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Selection Origin -&gt; Target</string>
             </property>
            </widget>
           </item>
           <item row="5" column="1">
            <widget class="QLabel" name="labelInfoTragetLayer">
             <property name="font">
              <font>
               <family>Droid Sans [unknown]</family>
               <italic>true</italic>
              </font>
             </property>
             <property name="text">
              <string>Select target layer from combobox</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelOriginLayerSelect">
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Origin Layer:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QComboBox" name="originLayerSelect">
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelTargetLayerSelect">
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Target Layer:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QComboBox" name="targetLayerSelect">
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageSelectOperation">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QLabel" name="labelTitleStep2">
             <property name="font">
              <font>
               <family>Droid Sans [unknown]</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Selection Geometry operation</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="geoprocessingIntersectRadio">
             <property name="inputMethodHints">
              <set>Qt::ImhNone</set>
             </property>
             <property name="text">
              <string>Intersection</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="geoprocessingTouchRadio">
             <property name="text">
              <string>Touch</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QRadioButton" name="geoprocessingContainRadio">
             <property name="text">
              <string>Contain</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageFieldsSelector">
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QLabel" name="labelTitleStpe3">
             <property name="font">
              <font>
               <family>Droid Sans [unknown]</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Selection fields</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelOutputLayerFields">
             <property name="text">
              <string>Output fields</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelOriginLayerFields">
             <property name="text">
              <string>Origin layer fields</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="labelTargetLayerFields">
             <property name="text">
              <string>Target layer fields</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QListWidget" name="listWidgetOriginLayerFields">
             <property name="acceptDrops">
              <bool>true</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::NoDragDrop</enum>
             </property>
             <property name="viewMode">
              <enum>QListView::ListMode</enum>
             </property>
             <property name="batchSize">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QListWidget" name="listWidgetOutputLayerFields">
             <property name="dragDropOverwriteMode">
              <bool>true</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::DragDrop</enum>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QListWidget" name="listWidgetTargetLayerFields">
             <property name="acceptDrops">
              <bool>false</bool>
             </property>
             <property name="dragDropMode">
              <enum>QAbstractItemView::NoDragDrop</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageSelectOutput">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="labelTitleStep4">
             <property name="font">
              <font>
               <family>Droid Sans [unknown]</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Selection Output</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageRun">
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="4" column="0">
            <widget class="QProgressBar" name="progressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="labelProgress">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelTitleStep5">
             <property name="font">
              <font>
               <family>Droid Sans [unknown]</family>
               <pointsize>18</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Run</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTextBrowser" name="runtimeStepBrowser"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="forwardButton">
         <property name="text">
          <string>Forward</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="backButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="testButton">
         <property name="text">
          <string>TEST</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Save/Load</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QLabel" name="labelConfigInoutFile">
         <property name="text">
          <string>ConfigFile Load</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="configFileInput"/>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="browseConfigFileInputButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="labelConfigOutputFile">
         <property name="text">
          <string>ConfigFile Save</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="configFileOutput"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="browseConfigFileOutputButton">
         <property name="text">
          <string>Browse</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
